OpenSourceOrg/dotOrg
# Event Manager - mobile styles (dev)

#### Changes proposed in this Pull Request

* Added mobile styles to the Event Single page

#### Testing instructions

* Open an Event page from the frontend
* Resize the page (including mobile resolutions) and confirm that the cover image gets resized properly ([reference](https://github.com/a8cteam51/opensource-net/issues/49#issuecomment-2123167161))

Mentions https://github.com/a8cteam51/opensource-net/issues/49
